packages/storybook/package.json

```diff
@@ -17,5 +17,8 @@
     "@storybook/addon-links": "^5.3.17",
     "@storybook/addons": "^5.3.17",
     "@storybook/react": "^5.3.17"
+  },
+  "dependencies": {
+    "@backstage/theme": "^0.1.1-alpha.3"
```

Comment:
is this actually needed? O_o

Reply:
@Rugvip without it the running Storybook locally does not work. Feel free to verify on your end!

Reply:
Does it work for you?

Reply:
@Rugvip this was fixed in #539 so I’ll remove this bit.